lib/jsonld_serializers.js

```diff
@@ -109,6 +109,10 @@ class JsonLdItemSerializer extends JsonLdSerializer {
       }
     })
 
+    // Add hasVol and hasDate properties
+    stmts.hasItemVolumes = this.body.numItemVolumesParsed / this.body.numItems > process.env.BIB_HAS_VOLUMES_THRESHOLD
+    stmts.hasItemDates = this.body.numItemDatesParsed / this.body.numItems > process.env.BIB_HAS_DATES_THRESHOLD
+
```

Comment:
These need to live in the `ResourceSerializer` class, not the `JsonLdItemSerializer` base class to ensure that the properties are only calculated at the "resource" (i.e. bib) level. (Otherwise they'll be calculated for items also)

Reply:
💯